From c1c3c323874151bbb2f302ed36377cb74b3241aa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kondor=20D=C3=A1niel?= <kondor.dani@gmail.com>
Date: Sat, 4 May 2024 16:13:30 +0200
Subject: [PATCH] build: ensure that symbols used by plugins are available

---
 src/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index b126b637..83df531a 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -41,6 +41,8 @@ link_directories(
 add_executable (${PROJECT_NAME}
 	${cairo_dock_SRCS} )
 
+set_target_properties (${PROJECT_NAME} PROPERTIES ENABLE_EXPORTS True)
+
 # Link the executable to the librairies.
 target_link_libraries (${PROJECT_NAME}
 	${PACKAGE_LIBRARIES}
